frontend/src/components/Menu/MenuDialog.tsx

```diff
@@ -66,7 +67,8 @@ export const MenuDialog: FC<MenuDialogProps> = ({
     url: {
       required: t("message.url_is_invalid"),
       validate: (value: string = "") =>
-        isAbsoluteUrl(value) || t("message.url_is_invalid"),
+        (isAbsoluteUrl(value) && URL_REGEX.test(value)) ||
```

Comment:
Thank you @devamitranjan 🙏 

Although it looks good, I suggest we use rather the URL class : 

```
function isAbsoluteUrl(value: string) {
  let url;
  
  try {
    url = new URL(value);
  } catch (_) {
    return false;  
  }

  return url.protocol === "http:" || url.protocol === "https:";
}
```

Let me know what you think.

Reply:
@marrouchi Thanks for the suggestion. Did the changes.